R/module-helpers.R

```diff
@@ -353,133 +353,3 @@ get_ssp_regions <- function(pcGDP, reg_names, income_group,
 
   regions$region
 }
-
```

Comment:
I was thinking the exact same thing!... Well beyond my git-fu. I'll write the new branch and PR first, then we can delete this horrible thing.